### DISCUSSION TITLE
Bluetooth toggle on/off for Plugins/Flow.Launcher.Plugin.Sys

### DISCUSSION THREAD

**Discussion by dharrison4** [Flow-Launcher/Flow.Launcher#2179] [Plugin Request]
**Toggle**

A simple on/off toggle. No need for connecting or disconnecting specific Bluetooth devices, although that would be nice to have. The on/off is the core feature I'd love to be able to use in FL.

**Alternatives considered**

I don't see any other plugin alternatives. There's a plug-in for changing the sound devices but not manipulating Bluetooth functionality

**Comment by jjw24** [comment]
Moving this request to discussion.
